tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/HardcodedConfigurationStore.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Sdk.Tools.CheckEnforcer
+{
+    public class HardcodedConfigurationStore : IConfigurationStore
+    {
+        public int ApplicationID => 40233;
+        public string ApplicationName => "check-enforcer";
+        public int ApplicationTokenLifetimeInMinutes => 10;
+
+        public int MinimumCheckRuns => 1; // TODO: Need to make this config file driven.
```

Comment:
We should also add some exclusions to the configuration. So we can exclude things like the cla check alone allowing folks to pass the checks.

Reply:
Good idea. Adding the configuration file support is something I'll iron out this week. I'm thinking of a configuration file that looks like this:

```yaml
format: v0.1-alpha # version specifier for the config file.
enabled: [true | false] # global on-off-switch (reads from target branch)
minimumChecks: 1
rules:
- ignore: cla/license
- require: java - core - ci
  pattern: sdk/core
```

Reply:
... but we might start with this:

```yaml
format: v0.1-alpha # version specifier for the config file.
enabled: [true | false] # global on-off-switch (reads from target branch)
minimumChecks: 1
rules:
- ignore: cla/license
```

Which basically means - there needs to be at least one check, excluding cla/license. And if there is multiple checks they must all pass.

Reply:
OK - I now support a basic configuration file - so I think what we have here will now work for us. We can evolve as necessary. Seems to be pretty reliable and quick too but I guess we'll see how it scales.

Reply:
>- require: java - core - ci
  pattern: sdk/core # would look to have full glob support here

I'd like us to be careful about duplicating this filtering logic between this config as well as ci.yml files. It woiuld be great if we can share them in someway.